components-compose/src/main/java/uk/gov/hmrc/components/compose/ui/theme/Theme.kt

```diff
@@ -90,6 +122,18 @@ object HmrcTheme {
     val typography: HmrcTypography
         @Composable
         get() = LocalHmrcTypography.current
+
+    val dimensions: Dimensions
+        @Composable
+        get() = LocalAppDimens.current
+
+    val orientation: Orientation
+        @Composable
+        get() = LocalOrientationMode.current
+
+    val fonScale: Float
```

Comment:
```suggestion
    val fontScale: Float
```

Reply:
My mistake, I have renamed the variable name.